src/ducks/userInfo/userInfoActions.jsx

```diff
@@ -2,6 +2,8 @@ import axios from 'axios';
 
 export const USER_SIGN_UP = "USER_SIGN_UP";
 export const USER_SIGN_IN = "USER_SIGN_IN";
+export const AUTH_ERROR = "AUTH_ERROR";
```

Comment:
nitpick: I would name this action name same as others - starting from `USER_`.
I could be something like user_request_error